content/en/docs/use-cases/aws-codecommit.md

```diff
@@ -0,0 +1,192 @@
+weight: 10
+---
+
+## Cluster Creation
+
+The following creates an EKS cluster with some minimal configuration that will work well with Flux:
+
+```sh
+eksctl create cluster
+```
+
+{{% alert color="info" %}}
+When working with `eksctl` you can provide a `ClusterConfig`, using `-f`, to setup how you want your cluster to be
+defined. e.g. `eksctl create cluster -f cluster.yaml`
+
+```yaml
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+
+metadata:
+  name: basic-cluster
+  region: eu-north-1
+
+nodeGroups:
+- name: ng-1
+  instanceType: m5.large
+  desiredCapacity: 10
+- name: ng-2
+  instanceType: m5.xlarge
+  desiredCapacity: 2
+```
+{{% /alert %}}
+
+## Flux Installation for AWS CodeCommit
+
+Ensure you can login to [console.aws.amazon.com](https://console.aws.amazon.com) for your proper organization,
+and create a new repository to hold your Flux install and other Kubernetes resources.
+
+Clone the Git repository locally:
+
+```sh
+git clone ssh://git-codecommit.<region>.amazonaws.com/v1/repos/<my-repository>
+cd my-repository
+```
+
+Create a directory inside the repository:
+
+```sh
+mkdir -p ./clusters/my-cluster/flux-system
+```
+
+Download the [Flux CLI](../installation/_index.md#install-the-flux-cli) and generate the manifests with:
+
+```sh
+flux install \
+  --export > ./clusters/my-cluster/flux-system/gotk-components.yaml
+```
+
+Commit and push the manifest to the master branch:
+
+```sh
+git add -A && git commit -m "add components" && git push
+```
+
+Apply the manifests on your cluster:
+
+```sh
+kubectl apply -f ./clusters/my-cluster/flux-system/gotk-components.yaml
+```
+
+Verify that the controllers have started:
+
+```sh
+flux check
+```
+
+Create a `GitRepository` object on your cluster by specifying the SSH address of your repo:
+
+```sh
+flux create source git flux-system \
+  --git-implementation=libgit2 \
+  --url=ssh://git-codecommit.<region>.amazonaws.com/v1/repos/<my-repository> \
+  --branch=<branch> \
+  --ssh-key-algorithm=rsa \
+  --ssh-rsa-bits=4096 \
+  --interval=1m
+```
+
+The above command will prompt you to add a deploy key to your repository, but AWS CodeCommit
+does not support repository or org-specific deploy keys. You may add the deploy key to a user's
+personal SSH keys, but take note that revoking the user's access to the repository will
+also revoke Flux's access. The better alternative is to create a machine-user whose sole purpose is
+to store credentials for automation. Using a machine-user also has the benefit of being able to be read-only or
+restricted to specific repositories if this is needed.
+
+{{% alert color="info" %}}
+Unlike `git`, Flux does not support the ["shorter" scp-like syntax for the SSH
+protocol](https://git-scm.com/book/en/v2/Git-on-the-Server-The-Protocols#_the_ssh_protocol)
+(e.g. `git-codecommit.<region>.amazonaws.com:v3`).
+Use the [RFC 3986 compatible syntax](https://tools.ietf.org/html/rfc3986#section-3) instead: `git-codecommit.<region>.amazonaws.com/v3`.
```

Comment:
Here is `amazonaws.com/v3` but above is `amazonaws.com/v1`, which one is it?

Reply:
It's `v1`. Will update it now :+1: